app/assets/stylesheets/src/shared/_scaffolds.scss

```diff
@@ -1,5 +1,6 @@
 $footer-min-height: 117px;
 $navbar-min-height: 56px;
+$visited-link-color: #044e92;
```

Comment:
Move that color to `_colors.scss`, please

Reply:
Sure!